extensions/wikia/PageShare/scripts/init.js

```diff
@@ -0,0 +1,34 @@
+require(['wikia.window', 'wikia.tracker', 'jquery'], function(win, tracker, $) {
+	'use strict';
+
+	var trackFunc = tracker.buildTrackingFunction({
+		action: Wikia.Tracker.ACTIONS.CLICK,
+		category: 'share',
+		trackingMethod: 'both'
+	});
+
+	/**
+	 * @desc Share click handler
+	 *
+	 * @param {Event} event
+	 */
+	function shareLinkClick(event) {
+		event.stopPropagation();
+		event.preventDefault();
+
+		var service = $(event.target).closest('a'),
+			url = service.prop('href'),
+			title = service.prop('title'),
+			h = (win.innerHeight / 2 | 0), // round down
+			w = (win.innerWidth / 2 | 0);  // round down
+
+		trackFunc({label: service.data('share-service')});
```

Comment:
`data-share-service` is present in `<li class="page-share page-share-{{name}}" data-share-service="{{name}}">` and here `service` variable is an `a` element. Shouldn't it be `$(event.target).closest('li')` in line 19?